packages/@react-spectrum/table/src/TableView.tsx

```diff
@@ -824,46 +836,52 @@ function ResizableTableColumnHeader(props) {
               <div className={classNames(styles, 'spectrum-Table-headerCellText')}>{column.rendered}</div>
             }
             {
-              columnProps.allowsResizing && <ChevronDownMedium UNSAFE_className={classNames(styles, 'spectrum-Table-menuChevron')} />
+              (columnProps.allowsResizing || hasCustomActions) && <ChevronDownMedium UNSAFE_className={classNames(styles, 'spectrum-Table-menuChevron')} />
             }
           </TableColumnHeaderButton>
-          <Menu onAction={onMenuSelect} minWidth="size-2000" items={items}>
-            {(item) => (
-              <Item>
-                {item.label}
-              </Item>
-            )}
-          </Menu>
-        </MenuTrigger>
-        <Resizer
-          ref={resizingRef}
-          column={column}
-          showResizer={showResizer}
-          onResizeStart={onResizeStart}
-          onResize={onResize}
-          onResizeEnd={onResizeEnd}
-          triggerRef={useUnwrapDOMRef(triggerRef)} />
-        <div
-          aria-hidden
-          className={classNames(
-            styles,
-            'spectrum-Table-colResizeIndicator',
+          <Menu {...menuProps} onAction={onMenuSelect} minWidth="size-2000">
             {
-              'spectrum-Table-colResizeIndicator--visible': resizingColumn != null,
-              'spectrum-Table-colResizeIndicator--resizing': resizingColumn === column.key
+              hasPriorityActions && priorityActions
```

Comment:
Does the priority actions need a wrapping section? Maybe not necessary since there isn't a particular theme grouping sorting + resizing and/or resizing might be standalone...

Reply:
Nope, first Section doesn't contribute a Divider and I don't have a good title for a section of 'sorting' and 'resizing'